docs/rfcs/016-user-and-database-management.md

```diff
@@ -0,0 +1,68 @@
+## Zentih user and database management RFC
+
+### CREATE USER
+
+CREATE USER is only allowed from console, 
+because it requires generation of the access token and other complex logic.
+
+Client has to chose which databases will be accessible by the new user.
+Along with database access, we explicitly grant CREATE ON DATABASE privilege to the user.
+`GRANT CREATE ON DATABASE .. TO .. ;`
+to let the user to create extensions.
+
+All new users are granted with CREATEDB privilege by default.
+`CREATE ROLE .. WITH CREATEDB`
+Later we may add an option to manage it.
+
+*Q: Are we going to allow user to create new users and NOLIGIN roles to manage privileges?
+*`CREATE ROLE .. WITH CREATEROLE`
```

Comment:
I think no, at least for now. NOLOGIN roles are not involved in the auth process, but this may complicate things. So I'd postpone this until we got N feature requests about more complex roles management